.github/semantic.yml

```diff
@@ -28,4 +28,4 @@ scopes:
     - sma
     - deps
     - all
-
+    - core-keeper
```

Comment:
add `keeper` as well

Reply:
added in https://github.com/edgexfoundry/edgex-go/pull/5331/commits/559c4c90137a8ccfc1103e5d33e37a3750c153e1